eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -311,6 +311,7 @@ stages:
         container: ${{ variables.ubuntu2204Container }}
         targetOS: linux
         targetArchitecture: x64
+        useDevVersions: true
```

Comment:
Please add comment explaining why this uses dev versions (I assume CI protection) and link to a tracking issue to eventually change that back when we produce our official bits from the VMR.

Reply:
Are we going to ship assets from this leg? I assumed that this leg came from before we had the Mariner cross build legs (that actually produce assets that we'd ship).